workflow/airflow/README.md

```diff
@@ -0,0 +1,44 @@
+# Airflow Example Usage
+
+Dependencies
+------------
+
+* Python >=3.6
+* [pipenv](https://docs.pipenv.org/)
+
+Installation
+------------
+This will setup your virtual environment and install the required dependencies:
+
+`pipenv --three install`
+
+Set the `AIRFLOW_HOME` environment variable in the `.env` file to your absolute path:
+
+e.g. `AIRFLOW_HOME=~/...../walking-skeleton/workflow/airflow`
+
+Activate your virtual environment:
+
+`pipenv shell`
+
+Initialize the database:
+
+`airflow initdb`
+
+This step will setup an `airflow.db` `sqlite` file, your `airflow.cfg` file and your `unittests.cfg` file.
+ 
+Usage
+-----
+
+Activate your virtual environment:
+
+`pipenv shell`
+
+Start a scheduler instance in it's own process:
+
+`airflow scheduler`
```

Comment:
are there worker processes to run or does the `scheduler` work by itself?

Reply:
With the current setup (basically only suitable for local testing and very lightweight use) it uses a `SequentialExecutor` by default which will only run task instances sequentially (no workers needed). In production, we would probably use its `CeleryExecutor` which will require `rabbitmq` and `redis` and then it would need its own workers for parallel task processing.

Reply:
a `CeleryExecutor` should be part of the walking skeleton, but not of this PR